Makefile

```diff
@@ -52,20 +52,18 @@ deps: ## Ensures fresh go.mod and go.sum.
 
 .PHONY: generate
 generate: ## Generate all dynamic files.
-generate: generate-pow generate-sol
+generate: pkg/pow/kernelSource.go pkg/contracts/tellor/tellor.go pkg/contracts/getter/getter.go
```

Comment:
this means: run targets 
 - pkg/pow/kernelSource.go 
 - pkg/contracts/tellor/tellor.go 
 - pkg/contracts/getter/getter.go